### DISCUSSION TITLE
[Feature] Option to prevent trying to iidentify items flagged as organized

### DISCUSSION THREAD

**Discussion by mveinot** [stashapp/stash#4711] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Using StashDB to identify scenes processes all scenes, even ones already identified. This makes the process take longer and more importantly, puts undue strain on the stash DB api server.

**Describe the solution you'd like**
Add an option (enabled by default?) not to try to identify scenes that are already marked as organized. This should be an option, because perhaps someone updates a scene entry and occasionally it might be nice to pull in the latest information for all items

**Describe alternatives you've considered**
I don't believe there is an alternative means to accomplish this currently

**Additional context**
Maybe using the "organized" flag isn't the correct approach, maybe a new flag called "identified" in case some people don't want to conflate the organized flag with identified scenes.

**Comment by DedTV** [comment]
I believe this already the default behavior when using the "Identify" task. At least that's how it's always worked for me. When running the Identify task on my DB, it only scrapes the ~1500 unorganized scenes out of my ~43,000 file library.

To get scenes marked Organized to scrape, I have either use tags and the bulk scraper plugin, run 'UPDATE scenes 
SET organized = 0;' on my DB to clear the Organized flag or, manually select the Organized scenes I want to re-scrape and run Identify through the Scenes UI or scrape them via the Tagger (both of which are not affected by the Organized flag).

**Comment by stg-annon** [comment, marked as answer]
correct, Identify will ignore scenes flagged as "Organized" you can even have identify mark the scene as organized if you only want it to touch scenes once

**Comment by mveinot** [comment]
Is that so, maybe I just have so many unrecognized scenes that it seems like it's most of them

**Comment by mveinot** [comment]
> I believe this already the default behavior when using the "Identify" task. At least that's how it's always worked for me. When running the Identify task on my DB, it only scrapes the ~1500 unorganized scenes out of my ~43,000 file library.
> 
> 
> 
> To get scenes marked Organized to scrape, I have either use tags and the bulk scraper plugin, run 'UPDATE scenes 
> 
> SET organized = 0;' on my DB to clear the Organized flag or, manually select the Organized scenes I want to re-scrape and run Identify through the Scenes UI or scrape them via the Tagger (both of which are not affected by the Organized flag).

Fair enough. I will close this request!
